public-ui/src/app/data-browser/components/tooltip/tooltip-react.component.tsx

```diff
@@ -0,0 +1,82 @@
+import {
+  AfterViewInit,
+  Component,
+  ElementRef,
+  EventEmitter,
+  Input,
+  Output,
+  ViewChild,
+  ViewEncapsulation
+} from '@angular/core';
+import * as React from 'react';
+import { FunctionComponent } from 'react';
+import * as ReactDOM from 'react-dom';
+import { BaseReactWrapper } from '../../../data-browser/base-react/base-react.wrapper';
+import { triggerEvent } from '../../../utils/google_analytics';
+import { getTooltip, tooltips } from '../../services/tooltip.service';
+
+const containerElementName = 'root';
+
+const ClrIcon = ({className = '', ...props}) => {
```

Comment:
Nit: I think this is fine here for now but may want to make it reusable in a utils file or something when we start converting more components with Clarity icons

Reply:
Thanks agree. Moved it to be re-used.